server/src/domain/server-info/server-info.service.ts

```diff
@@ -93,6 +93,8 @@ export class ServerInfoService {
   }
 
   async getStatistics(): Promise<ServerStatsResponseDto> {
+    await this.userRepository.syncUsage();
```

Comment:
I think this endpoint is called every x seconds and I'm not 100% if it is a good idea to synchronize the usage _every time_ that happens.

Reply:
Should we run it nightly instead?

Reply:
We do already run it nightly, the idea was to prevent it from being out of sync when you specifically go to the admin page to look at user's usage, but I think it kicks off a timer the refreshes it every 5 seconds.